modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -96,9 +96,58 @@ class SegmentEdit extends Component {
 			values,
 		} = props;
 
+		// TODO: remove this when the bakend send the property group for events
+
+		const fakePropertyGroups = [
+			{
+				entityName: 'Event',
+				name: 'Event',
+				properties: [
+					{
+						label: 'Downloaded Documents and Media',
+						name: 'downloadedDocuments',
+						options: [],
+						selectEntity: null,
+						type: 'event',
+					},
+				],
+				propertyKey: 'event',
+			},
+		].concat(propertyGroups);
+
+		const fakeInitialContributors = initialContributors.concat({
+			conjunctionId: 'and',
+			conjunctionInputId:
+				'_com_liferay_segments_web_internal_portlet_SegmentsPortlet_criterionConjunctionevent',
+
+			initialQuery: {
+				conjunctionName: 'and',
+				groupId: 'group_0',
+				items: [
+					{
+						assetId: 'documentId',
+						day: {
+							operatorName: 'between',
+							value: {
```

Comment:
Also just in case, a reminder of the dates, since here you are using dates with time values. I've tried some cases without the time and still working, also in the cases that doesn't use ranges, but maybe you want to take a look again.